src/components/vm/overview/descriptionModal.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+import cockpit from 'cockpit';
+import { Button } from "@patternfly/react-core/dist/esm/components/Button/index.js";
+import { Form, FormGroup } from "@patternfly/react-core/dist/esm/components/Form/index.js";
+import { TextInput } from "@patternfly/react-core/dist/esm/components/TextInput/index.js";
+import { Modal } from "@patternfly/react-core/dist/esm/components/Modal/index.js";
+import { useDialogs } from 'dialogs.jsx';
+
+import { ModalError } from 'cockpit-components-inline-notification.jsx';
+import { domainSetDescription } from '../../../libvirtApi/domain.js';
+
+const _ = cockpit.gettext;
+
+export const DescriptionModal = ({ vm }) => {
+    const Dialogs = useDialogs();
+    const [error, setError] = useState({});
+    const [newDescription, setDescription] = useState(vm.description);
```

Comment:
Just a nitpick - but we like to use the same name for the variable as the setter. Just use const [description, setDescription] = ...

Reply:
Makes sense. I will change it,  but i will use `[newDescription, setNewDescription]` to be consistent with the rename dialog code.